release-notes/3.0/preview/api-diff/preview6/3.0-preview6_System.Numerics.md

```diff
@@ -0,0 +1,25 @@
+# System.Numerics
+
+``` diff
+ namespace System.Numerics {
```

Comment:
`System.Numerics` LGTM.

Also CC. @GrabYourPitchforks since he implemented the Span overloads on `Vector<T>`.